web/beacon-app/src/features/members/api/_tests_/memberList.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { vi } from 'vitest';
+
+import { memberRequest } from '../memberListAPI';
+
+describe('Member', () => {
+  describe('Member List', () => {
+    it('returns member list request resolved with response', async () => {
+      const mockMembersResponse = {
+        PromiseRejectionEvent: [
+          {
+            id: '1',
+            name: 'Kamala Khan',
+            role: 'Owner',
+          },
+        ],
+      };
+
+      const requestSpy = vi.fn().mockReturnValueOnce({
+        status: 200,
+        data: mockMembersResponse,
+        statusText: 'OK',
+      });
+      const request = memberRequest(requestSpy);
+      const response = await request();
+      expect(response).toBe(mockMembersResponse);
+      expect(requestSpy).toHaveBeenCalledTimes(1);
+    });
+  });
+});
```

Comment:
We should add more cases but because we are running out of time we could review ait in the next sprint by adding failure cases.

Reply:
I'll create a spike story in sprint 4 about adding test cases.